### PR TITLE
AG-12640 - Hide maps example titles

### PR DESCRIPTION
**Pull request by markdurrant**

### CODE REVIEW

File: packages/ag-charts-website/src/pages-styles/homepage.module.scss [hunk 1]
```diff
@@ -522,12 +522,26 @@
...
+
     :global(.example-runner-outer) {
-        margin-bottom: 0 !important;
+        margin-top: $spacing-size-2;
+        margin-bottom: $maps-title-offset !important;
     }
 
     :global(div[class*='content']) {
-        border: none;
+        overflow: hidden;
```
Comment (taktran) [inline packages/ag-charts-website/src/pages-styles/homepage.module.scss:539]: I think there's already another one above, where there is a `border: none;`

**Review by taktran — APPROVED** [review]
Just a minor comment

**Comment by JamesSwinton** [comment]
LGTM

File: packages/ag-charts-website/src/pages-styles/homepage.module.scss (hunk 1 above)
Comment (markdurrant) [reply packages/ag-charts-website/src/pages-styles/homepage.module.scss:539]: This is intentional to keep all the quick fix stuff separate from other things.

**Pull request merged**